libevm/VMOpt.cpp

```diff
@@ -57,20 +57,27 @@ void VM::copyCode(int _extraBytes)
 	auto extendedSize = m_ext->code.size() + _extraBytes;
 	m_codeSpace.reserve(extendedSize);
 	m_codeSpace = m_ext->code;
-	m_codeSpace.resize(extendedSize);
+	resizeCodeSpace(extendedSize);
+}
+
+void VM::resizeCodeSpace(size_t _size)
+{
+	m_codeSpace.resize(_size);
 	m_code = m_codeSpace.data();
 }
 
```

Comment:
No, it was just extracting these two lines into separate method, since it was needed twice. But I'll remove it since you say it's not needed, thanks